Hydra/EventListener/RequestExceptionListener.php

```diff
@@ -25,52 +20,19 @@
  * @author Samuel ROZE <samuel.roze@gmail.com>
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class RequestExceptionListener
+class RequestExceptionListener extends ExceptionListener
 {
-    const FORMAT = 'jsonld';
-
-    /**
-     * @var NormalizerInterface
-     */
-    private $normalizer;
-
-    public function __construct(NormalizerInterface $normalizer)
-    {
-        $this->normalizer = $normalizer;
-    }
-
     /**
      * @param GetResponseForExceptionEvent $event
      */
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
+        // Normalize exceptions with hydra errors only for resources
         $request = $event->getRequest();
-        if (!$request->attributes->has('_resource_type') || self::FORMAT !== $request->attributes->get('_api_format')) {
+        if (!$request->attributes->has('_resource_type')) {
```

Comment:
`$event->getRequest()` could be inlined here IMO (as it is in 1.0), as it is the unique usage here and is still pretty readable.